doc/STATE_ROLES.md

```diff
@@ -68,7 +68,9 @@
 * value.lock            - actual position of lock
 * value.max
 * value.min
-* value.power.consumption (unit=Wh or KWh)
+* value.power.consumption   (unit=Wh or KWh)
```

Comment:
Nitpicking, but the correct unit would be `kWh`, not `KWh` :grimacing:

Reply:
yeah aggree, fixed